src/test/lrsql/util/oidc_test.clj

```diff
@@ -0,0 +1,16 @@
+(ns lrsql.util.oidc-test
+  (:require [clojure.test :refer :all]
+            [lrsql.util.oidc :refer :all]))
```

Comment:
Oh dear VSCode is going to scream at these `:all`s.

Reply:
Why? Is this a kondo thing? That is practically idiomatic in clojure tests

Reply:
It is indeed a kondo thing. Granted you can suppress it using `#_{:clj-kondo/ignore [:refer-all]}` but otherwise it'll be annoying.